pkg/auth/authz_handler.go

```diff
@@ -19,34 +20,19 @@ import (
 	"gorm.io/gorm"
 )
 
+/* AuthzHandler is the authorizer middleware that is being used for
+ * RHSSO authentication cases and work with AMS to ensure tenancy
+ * and user based access policies
+ */
 type AuthzHandler struct {
```

Comment:
maybe worth renaming to RHSSOAuthorizer, i.e. to be consistent with RHSSOAuthenticator (also the file)